scripts/hail_batch/hgdp1kg_tobwgs_nfe/hgdp_1kg_tob_wgs_nfe.py

```diff
@@ -0,0 +1,59 @@
+"""Perform pca on nfe samples from the HGDP,1KG, and tob-wgs dataset"""
+
+import click
+import pandas as pd
+import hail as hl
+
+HGDP1KG_TOBWGS = (
+    'gs://cpg-tob-wgs-analysis/1kg_hgdp_tobwgs_pca/'
+    'v1/hgdp1kg_tobwgs_joined_all_samples.mt'
+)
+
+GNOMAD_LIFTOVER_LOADINGS = 'gs://cpg-reference/gnomad/gnomad_loadings_90k_liftover.ht'
+
+GNOMAD_HGDP_1KG_MT = (
+    'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/'
+    'gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'
+)
+
+
+@click.command()
+@click.option('--output', help='GCS output path', required=True)
+def query(output):  # pylint: disable=too-many-locals
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    mt = hl.read_matrix_table(HGDP1KG_TOBWGS)
+    mt = mt.annotate_cols(TOB_WGS=mt.s.contains('TOB'))
+    # Get NFE samples only
+    mt = mt.filter_cols(
+        (mt.hgdp_1kg_metadata.population_inference.pop == 'nfe') | (mt.TOB_WGS)
+    )
+    mt_path = f'{output}/hgdp1kg_nfe_samples.mt'
+    if not hl.hadoop_exists(mt_path):
+        mt.write(mt_path)
```

Comment:
Should this be a checkpoint to the `temporary` bucket or is this actually useful to keep a copy of?

Reply:
I thought it would be useful to keep a copy of (for performing PCA later), but you're right in that it's not super useful downstream and easy to generate. Since it's so quick to filter the samples, do we even need a checkpoint?

Reply:
Good point, a checkpoint isn't really worth it either.

Reply:
great, will just leave it without a checkpoint/saving